lib/omnibus/packagers/bff.rb

```diff
@@ -229,6 +232,9 @@ def create_bff_file
       FileSyncer.glob(File.join(staging_dir, 'tmp/*.bff')).each do |bff|
         copy_file(bff, File.join(Config.package_dir, create_bff_file_name))
       end
+
+      # chown back to original user's uid/gid so cleanup works correctly
+      shellout!("sudo chown -Rh #{original_uid}:#{original_gid} #{staging_dir}")
```

Comment:
I think if something fails we still want to chown it back for cleanup, especially since this is all in a tmpdir and we will need to do a new build anyways :)